contents/docs/getting-started/data-import-export.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Data import and export
+---
+
+There are many ways to get data in and out of PostHog. In addition to the automated convenience of data pipeline sources and destinations, we’ve also got some manual options for bringing data in from other platforms.
```

Comment:
We should make it clearer here the this is beyond capturing data from your site or app with PostHog.